csv.rb

```diff
@@ -0,0 +1,16 @@
+file_name = "new.csv"
+
+file_content = []
+file = File.open(file_name, "r")
+while (line = file.gets)
+    file_content.push(line)
+end
+
+print file_content
+file = File.open(file_name, "w")
+for line in file_content
+    file.write("#{line.to_s} \n")
+end
```

Comment:
@mohseengs I do not see any csv file here? commit the csv file as well in this PR, and also this is not the correct code for reading a CSV file.